### ISSUE TITLE
Error: Loading chunk 0 failed.

### ISSUE THREAD

**Issue by davidhpark** [masotime/json-url#6]
```
GET http://localhost:3000/json-url-msgpack.js 404 (Not Found)

Refused to execute script from 'http://localhost:3000/json-url-msgpack.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.

json-url.js:1 Uncaught (in promise) Error: Loading chunk 0 failed.
    at HTMLScriptElement.e (json-url.js:1)
```
Trying to run the example and I ran into this issue. Is this related at all to #5?

**Comment by masotime** [comment]
Yeah Webpack integration is borked currently, let me look into it.

**Comment by masotime** [comment]
Hmm... the example should work though.

**Comment by masotime** [comment]
Are you running the example using `npm run example`? The web server you're using doesn't seem to be returning the right `MIME` type for Javascript files.

**Comment by davidhpark** [comment]
Is there a way to bypass the GET request the package makes and directly convert from the JSON to URL?

**Comment by masotime** [comment]
So I've started looking into this, but GDPR is consuming a lot of my time - still, I have a few ideas that may work out.

**Comment by Christilut** [comment]
Same problem here with Browserify, any good workaround for now?

**Comment by edanweis** [comment]
+1

**Comment by logonod** [comment]
+1

**Comment by username99987** [comment]
+1

**Comment by masotime** [comment]
@username99987 - I've updated the module with a single-bundle fix for Webpack for 2.4.0. Can you try it out?

**Comment by masotime** [comment]
Closing for now. Please re-open if the issue still persists.
